Neos.Neos/Classes/Command/CrCommandController.php

```diff
@@ -114,6 +119,9 @@ public function importCommand(string $path, string $contentRepository = 'default
         $projectionService = $this->contentRepositoryRegistry->buildService($contentRepositoryId, $this->projectionReplayServiceFactory);
         $projectionService->replayAllProjections(CatchUpOptions::create());
 
+        $this->outputLine('Assigning live workspace role');
+        $this->workspaceService->assignWorkspaceRole($contentRepositoryId, WorkspaceName::forLive(), WorkspaceRoleAssignment::createForGroup('Neos.Neos:LivePublisher', WorkspaceRole::COLLABORATOR));
```

Comment:
imo as hotfix we also should create a site entity for each imported site node here :)

Reply:
I'll take care of that in a follow-up